bin/pd-nagios

```diff
@@ -176,6 +185,18 @@ def parse_fields(fields):
         return {}
     return dict(f.split("=", 1) for f in fields)
 
+def parse_env_fields(details, env_fields):
+    if env_fields is None:
+        return
+    import os
+    for f in env_fields:
+        v = os.environ.get("NAGIOS_%s" % f)
```

Comment:
how about `v = os.environ.get("NAGIOS_%s" % f) or os.environ.get("ICINGA_%s" % f)` to simplify the logic a bit